doc/Learn/Tutorials/Navigation/Advanced/HowTo-UsePanel.md

```diff
@@ -1,23 +1,252 @@
+              uen:Navigation.Request="./Three" />
+  </StackPanel>
+  ```
+
+* This will automatically set up navigation to the corresponding view when the `Button.Click` event is raised.
+
+* Next, we need to attach content regions for each view. The first step is to set the `Region.Attached` attached property to `True` on the parent `Grid` containing the views:
+  
+  ```xml
+  <Grid Grid.Row="2" 
+        uen:Region.Attached="True">
+      <Grid>
+          <TextBlock Text="One" 
+                    FontSize="24" 
+                    HorizontalAlignment="Center" 
+                    VerticalAlignment="Center"/>
+      </Grid>
+      <Grid>
+          <TextBlock Text="Two"
+                    FontSize="24"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center" />
+      </Grid>
+      <Grid>
+          <TextBlock Text="Three"
+                    FontSize="24"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center" />
+      </Grid>
+  </Grid>
+  ```
+
+* Since we want to toggle visibility of the views, we need to set the `Region.Navigator` attached property to `Visibility`:
+
+  ```xml
+  <Grid Grid.Row="2" 
+        uen:Region.Attached="True"
+        uen:Region.Navigator="Visibility">
+      <Grid>
+          <TextBlock Text="One" 
+                    FontSize="24" 
+                    HorizontalAlignment="Center" 
+                    VerticalAlignment="Center"/>
+      </Grid>
+      <Grid>
+          <TextBlock Text="Two"
+                    FontSize="24"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center" />
+      </Grid>
+      <Grid>
+          <TextBlock Text="Three"
+                    FontSize="24"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center" />
+      </Grid>
+  </Grid>
+  ```
+
+* Finally, we need to set the `Region.Name` attached property on each child `Grid` to the corresponding view name:
+
+  ```xml
+  <Grid Grid.Row="2" 
+        uen:Region.Attached="True"
+        uen:Region.Navigator="Visibility">
+      <Grid uen:Region.Name="One">
+          <TextBlock Text="One" 
+                    FontSize="24" 
+                    HorizontalAlignment="Center" 
+                    VerticalAlignment="Center"/>
+      </Grid>
+      <Grid uen:Region.Name="Two">
+          <TextBlock Text="Two"
+                    FontSize="24"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center" />
+      </Grid>
+      <Grid uen:Region.Name="Three">
+          <TextBlock Text="Three"
+                    FontSize="24"
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center" />
+      </Grid>
+  </Grid>
+  ```
+
+* Now, you have written a UI layout capable of navigating to views with `Panel`. Your completed `MainPage.xaml` should look like the code example below.
```

Comment:
We should add a step here that sets the visibility of each of the regions to Collapsed